src/databricks/labs/remorph/install.py

```diff
@@ -21,80 +21,145 @@
     SpotInstancePolicy,
 )
 
+from databricks.labs.remorph.__about__ import __version__
+from databricks.labs.remorph.config import MorphConfig, ReconcileConfig, SQLGLOT_DIALECTS, DatabaseConfig
+from databricks.labs.remorph.reconcile.constants import SourceType
+
 logger = logging.getLogger(__name__)
 
 PRODUCT_INFO = ProductInfo(__file__)
 WAREHOUSE_PREFIX = "Remorph Transpiler Validation"
 
 
 class WorkspaceInstaller:
-    def __init__(self, prompts: Prompts, installation: Installation, ws: WorkspaceClient):
+    def __init__(self, ws: WorkspaceClient, prompts: Prompts = Prompts()):
         if "DATABRICKS_RUNTIME_VERSION" in os.environ:
             msg = "WorkspaceInstaller is not supposed to be executed in Databricks Runtime"
             raise SystemExit(msg)
         self._ws = ws
-        self._installation = installation
         self._prompts = prompts
         self._catalog_setup = CatalogSetup(ws)
+        self._product_info = ProductInfo.from_class(MorphConfig)
+        self._install = InstallPrompts(ws, prompts)
+
+    @cached_property
+    def _installation(self):
+        try:
+            return Installation.assume_user_home(self._ws, self._product_info.product_name())
+        except RuntimeError:
```

Comment:
Not sure we need global install, we can ignore.

Reply:
Removed